### ISSUE TITLE
Only works when loading config.ini

### ISSUE THREAD

**Issue by PokemonCrazy** [AHAAAAAAA/PokemonGo-Map#1892]
When I use -se to boot from the config.ini, it works fine, but if I type out any command, it says a password is required.
I try to run:
runserver.py -u [username] -p [password] -l "[x-coord] [y-coord]" -st 5 -dp -dg
With my username, password, and coordinates filled out, but runserver.py throws me this error every time.
runserver.py: error: argument -p/--password: expected one argument

**Comment by Smilhjan** [comment]
add `-a ptc` or `-a google` depending on your login method

**Comment by RyanMBlue** [comment]
I don't believe -dp and -dg are supported anymore. The extra p is throwing it off. Remove those arguments
